Lib/test/test_descr.py

```diff
@@ -1826,7 +1848,8 @@ def __init__(self, foo):
         object.__init__(A(3))
         self.assertRaises(TypeError, object.__init__, A(3), 5)
 
-    @unittest.expectedFailure
+    # RUSTPYTHON: this is added from cpython3.11.2. below test case is success in rustpython
```

Comment:
Minor clarification:
```suggestion
    # TODO: RUSTPYTHON, CPython 3.5 and above expect this test case to fail, but in RustPython this currently passes.
    # See https://github.com/python/cpython/issues/49572 for more details.
```